Lib/fontTools/ufoLib/glifLib.py

```diff
@@ -695,12 +695,12 @@ def _writeUnicodes(glyphObject, element, validate):
 
 def _writeNote(glyphObject, element, validate):
 	note = getattr(glyphObject, "note", None)
-	if validate and not isinstance(note, basestring):
-		raise GlifLibError("note attribute must be str or unicode")
+	if validate and not isinstance(note, str):
+		raise GlifLibError("note attribute must be str")
 	note = note.strip()
 	note = "\n" + note + "\n"
-	# ensure text is unicode, if it's bytes decode as ASCII
-	etree.SubElement(element, "note").text = tounicode(note)
+	# ensure text is str, if it's bytes decode as ASCII
```

Comment:
the comment is stale, you're no longer ensuring text is `str`, you just setting text attribute directly (assuming it's already a `str`, which it is). remove the comment